source/content/guides/drupal-9/drupal-9-hosted-pre112021/01-introduction.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Migrate a Site That Was Created Before November 2011 to Drupal 9
+subtitle: Introduction
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-pre112021
+anchorid: drupal-9-hosted-pre112021
+editpath: drupal-9/drupal-9-hosted-pre112021/01-introduction.md
+---
+
+This guide will show you how to migrate a site that meets the following criteria to Drupal 9:
+
+|Criteria|Value
+|---|---
+|**Pantheon Hosted**| **Yes**
+|Site created using:| n/a
+|Multi-Dev Environment | n/a
+|Build Tools Workflow Needed | n/a
+|Drupal version set to 8| n/a
+|**Site created before November 2021**| **Yes**
+
+Drupal 9 sites created on the platform prior to November 30, 2021 use the [Drupal 9](https://github.com/pantheon-upstreams/drupal-project) upstream. Based on community needs, we have released a new upstream. [Drupal with Composer](https://github.com/pantheon-upstreams/drupal-recommended) is now the default Drupal 9 upstream on the platform and users are encouraged to switch to it to take advantage of the improved structure and updates.
+
+
+## Requirements
+This guide requires [User in Charge](/change-management#site-level-roles-and-permissions) permissions to set the upstream.
```

Comment:
@wordsmither [User in Charge](https://pr-7235--pantheon-docs.my.pantheonfrontend.website/docs/change-management#roles-and-permissions) is present in the dashboard for Enterprise and EDU Organizations. I'm not sure what permission level is required for customers who aren't Enterprise or EDU Organizations, but I think it should probably be added here?

Reply:
I can speak to this. This is not needed anymore and should be removed :)

Reply:
@kporras07 Do you know which roles are allowed to perform this action? I think it would be helpful to our users to add this information.

Reply:
When this change was done, docs were updated: https://pantheon.io/docs/change-management It seems it's "everyone"?

Reply:
Thanks @kporras07